Makefile

```diff
@@ -94,6 +94,8 @@ TEST_LDFLAGS       = $(COMMON_LDFLAGS) -L$(objdir)/libtraceevent -ltraceevent
 
 ifeq ($(DEBUG), 1)
   COMMON_CFLAGS += -O0 -g -DDEBUG_MODE=1
+else ifeq ($(DEBUG), 2)
+  COMMON_CFLAGS += -O0 -gdwarf-2 -g3 -DDEBUG_MODE=1
```

Comment:
Is the `-gdwarf-2` really needed?

Reply:
Right. It's not needed so just removed it.